setup.py

```diff
@@ -98,6 +104,7 @@
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
         'Programming Language :: Python :: 3.11',
+        'Programming Language :: Python :: 3.12',
```

Comment:
🤔

Reply:
It passes :D

https://github.com/inspirehep/inspire-utils/actions/runs/9991909905/job/27615699247?pr=95